README.md

```diff
@@ -58,6 +69,23 @@ You can query Document nodes created from your Strapi API like the following:
 }
 ```
 
+You can query Document nodes in chosen language
+> make sure to add `api.qs._locale` to your strapi conf in `gatsby-config.js` (see example above)
```

Comment:
[Strapi conf](https://conf.strapi.io/) is its own thing, so I'd recommend typing "configuration" entirely

Also, I don't think this needs to be a quote (`>`), you can add this as text directly

And I would say "You can query Document nodes in *a* chosen language"